.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,16 +1,40 @@
-#### What does this PR do?
```

Comment:
I wouldn't totally ditch all written prose in the template.  I think that there is value in at least a bulleted list of changes/additions, or a few sentences describing the work.  But I agree that the there were too many sections here before.